code/game/objects/items/food/pizza.dm

```diff
@@ -13,6 +13,10 @@
 	///What label pizza boxes use if this pizza spawns in them.
 	var/boxtag = ""
 
+/obj/item/food/pizza/ComponentInitialize()
```

Comment:
componentinitialize is deprecated

Reply:
I am devastated by butterdog code lying to me